.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.7.9-stretch
+
+    steps:
+      - checkout
+
+      - run:
+          run: setup_creds
+          command: |
+            echo $GCLOUD_SERVICE_KEY | base64 --decode --ignore-garbage > ${HOME}/gcloud-service-key.json
+      - restore_cache:
+          key: deps2-{{ .Branch }}
+
+      - run:
+          name: "Setup dbt"
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install --upgrade pip setuptools
+            pip install -r integration_tests/requirements.txt
+            mkdir -p ~/.dbt
+            cp integration_tests/ci/sample.profiles.yml ~/.dbt/profiles.yml
+      - run:
+          name: "Run Tests - Redshift"
+          command: |
+            . venv/bin/activate
+            echo `pwd`
+            cd integration_tests
+            dbt deps
+            dbt seed --target redshift
+            dbt run --target redshift --full-refresh
+            dbt test --target redshift
+      - run:
+          name: "Run Tests - Snowflake"
+          command: |
+            . venv/bin/activate
+            echo `pwd`
+            cd integration_tests
+            dbt deps
+            dbt seed --target snowflake
+            dbt run --target snowflake --full-refresh
+            dbt test --target snowflake
+      - run:
+          name: "Run Tests - BigQuery"
+          environment:
+              GCLOUD_SERVICE_KEY_PATH: "/home/circleci/gcloud-service-key.json"
+
+          command: |
+            . venv/bin/activate
+            echo `pwd`
+            cd integration_tests
+            dbt deps
+            dbt seed --target bigquery
+            dbt run --target bigquery --full-refresh
+            dbt test --target bigquery
+      - save_cache:
+          key: deps2-{{ .Branch }}
+          paths:
+            - "venv"
```

Comment:
Example of a full config.yml
https://github.com/fivetran/dbt_iterable_source/blob/main/.circleci/config.yml